modular_citadel/code/modules/mob/living/silicon/robot/robot_movement.dm

```diff
@@ -18,6 +18,8 @@
 		if(sprinting)
 			playsound_local(src, 'modular_citadel/sound/misc/sprintactivate.ogg', 50, FALSE, pressure_affected = FALSE)
 		else
+			if(!(cell?.charge))
```

Comment:
Make sure this checks to see if cell charge is less than or equal to the amount taken by sprinting. Also, why does this not apply to the sprint sound toggle on as well?